### PR TITLE
help me understand the mechanisms

### PR DESCRIPTION
**Pull request by mfrasca**
I've altered where we first mention `logger` according to my understanding.

then, possibly one or two sections too far into the text, I realize there's questions to be posed and answered.  preferably by breaking things and showing what it is that we've been using, unaware of it.

### CODE REVIEW

File: getting-started/mix-otp/supervisor-and-application.markdown [hunk 1]
```diff
@@ -156,7 +156,9 @@ We can find the generated `.app` file at `_build/dev/lib/kv/ebin/kv.app`. Let's
               {extra_applications,[logger]}]}.
 ```
 
-This file contains Erlang terms (written using Erlang syntax). Even though we are not familiar with Erlang, it is easy to guess this file holds our application definition. It contains our application `version`, all the modules defined by it, as well as a list of applications we depend on, like Erlang's `kernel`, `elixir` itself, and `logger` which is specified in the `:extra_applications` list in `mix.exs`.
+This file contains Erlang terms (written using Erlang syntax). Even though we are not familiar with Erlang, it is easy to guess this file holds our application definition. It contains our application `version`, all the modules defined by it, as well as a list of applications we depend on, like Erlang's `kernel`, `elixir` itself, and `logger`. 
+
+> The `logger` application is started by default with Elixir. We stated that our application needs it by specifying it in the `:extra_applications` list in `mix.exs`.
```
Comment (josevalim) [inline getting-started/mix-otp/supervisor-and-application.markdown]:
```suggestion
> The `logger` application ships as part of Elixir. We stated that our application needs it by specifying it in the `:extra_applications` list in `mix.exs`. See the [official docs](https://hexdocs.pm/logger) for more information.
```

File: getting-started/mix-otp/supervisor-and-application.markdown [hunk 2]
```diff
@@ -215,7 +217,7 @@ Nothing really exciting happens but it shows how we can control our application.
 
 ## The application callback
 
-TODO: what do you think of reducing amount of words, distill content, only repeat when expanding the concepts.
+TODO: sorry, here I'm understanding less than beans from here on. I think that before we start modifying things, we should explain how things work in the first place. Like, how is it that the `Application.start(:kv)` tells us that `:kv` is `:already_started`? Where is that written in the code, or how is it assumed by the system? Is there anything we can change in `mix.exs`, or wherever else, that influences this? But I wish to see it in a single sentence, like "change this, and try it out". Oh, by the way, either I missed it, or we do not yet know how to reload modules, that is: compile and reload, from iex.
```
Comment (josevalim) [inline getting-started/mix-otp/supervisor-and-application.markdown]: Is this comment in  the correct place? The part we talk about Application.start(:kv) is the previous session.

Comment (mfrasca) [reply getting-started/mix-otp/supervisor-and-application.markdown]: this is the spot where I realized I could not any more follow the text.

**Comment by josevalim** [comment]
I have broken the flow down a bit. Ssince this already hass a couple changes, i will merge it, but please feel free to open up another issue!

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**